src/Sql/Sql/DataClassification/Cmdlet/GetAzSqlDatabaseSensitivityClassification.cs

```diff
@@ -0,0 +1,147 @@
+﻿using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
```

Comment:
@bashahee quick comment: any file that gets shipped with your module should contain a [Microsoft license header](https://github.com/Azure/azure-powershell/blob/master/src/Sql/Sql/Common/SqlSkuUtils.cs#L1-L13) -- please add this to all of the files you plan on shipping that are missing this